.env

```diff
@@ -0,0 +1,3 @@
+REACT_APP_APPLICATION_NAME=$npm_package_name
+REACT_APP_VERSION=$npm_package_version
```

Comment:
We can have first 2 variable in `.env.local`. Which will populate for all env, except `.env.test` will have to manually copy.